debian/copyright

```diff
@@ -0,0 +1,34 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: nemo-qml-plugin-dbus
+Source: https://git.sailfishos.org/mer-core/nemo-qml-plugin-dbus
+
+Files: *
+Copyright: 2013-2021 Jolla OY & many contributors (see git log)
+License: LGPL-2.1
```

Comment:
I think this should be like "BSD and LGPLv2" as there are both. This order is the same as in the rpm spec.

Reply:
I noticed that the rpm license is BSD and LGPLv2, and the debian/copyright license is also changed to the same